[0m[[0minfo[0m] [0mRunning tutorial.webapp.HelloWorldApp[0m
[0m[[0mdebug[0m] [0mwith JSEnv ExternalJSEnv for Node.js with JSDOM[0m
[0m[[0mdebug[0m] [0mStarting process: node[0m
[0m[[31merror[0m] [0mError: Could not load script: "c:\Users\trolo\AppData\Local\Temp\1489607117407-0\jquery.js"[0m
[0m[[31merror[0m] [0m    at e (D:\Prog\ScalaJS\NESemul\node_modules\jsdom\lib\jsdom\browser\resource-loader.js:44:23)[0m
[0m[[31merror[0m] [0m    at Object.check (D:\Prog\ScalaJS\NESemul\node_modules\jsdom\lib\jsdom\living\nodes\Document-impl.js:88:11)[0m
[0m[[31merror[0m] [0m    at ReadStream.<anonymous> (D:\Prog\ScalaJS\NESemul\node_modules\jsdom\lib\jsdom\living\nodes\Document-impl.js:107:12)[0m
[0m[[31merror[0m] [0m    at ReadStream.wrappedEnqueued (D:\Prog\ScalaJS\NESemul\node_modules\jsdom\lib\jsdom\browser\resource-loader.js:255:16)[0m
[0m[[31merror[0m] [0m    at emitOne (events.js:96:13)[0m
[0m[[31merror[0m] [0m    at ReadStream.emit (events.js:188:7)[0m
[0m[[31merror[0m] [0m    at fs.js:1941:12[0m
[0m[[31merror[0m] [0m    at FSReqWrap.oncomplete (fs.js:123:15) { Error: ENOENT: no such file or directory, open 'D:\Users\trolo\AppData\Local\Temp\1489607117407-0\jquery.js'[0m
[0m[[31merror[0m] [0m    at Error (native)[0m
[0m[[31merror[0m] [0m  errno: -4058,[0m
[0m[[31merror[0m] [0m  code: 'ENOENT',[0m
[0m[[31merror[0m] [0m  syscall: 'open',[0m
[0m[[31merror[0m] [0m  path: 'D:\\Users\\trolo\\AppData\\Local\\Temp\\1489607117407-0\\jquery.js' }[0m
[0m[[31merror[0m] [0m[0m
[0m[[31merror[0m] [0md:\Prog\ScalaJS\NESemul\target\scala-2.11\nesemul-fastopt.js:1530[0m
[0m[[31merror[0m] [0m  (0, $m_Lorg_scalajs_jquery_package$().jQuery$1)((function() {[0m
[0m[[31merror[0m] [0m                                                 ^[0m
[0m[[31merror[0m] [0mTypeError: (0 , $m_Lorg_scalajs_jquery_package$(...).jQuery$1) is not a function[0m
[0m[[31merror[0m] [0m    at $c_Ltutorial_webapp_HelloWorldApp$.main__V (d:\Prog\ScalaJS\NESemul\target\scala-2.11\nesemul-fastopt.js:1530:50)[0m
[0m[[31merror[0m] [0m    at $c_Ltutorial_webapp_HelloWorldApp$.$$js$exported$meth$main__O (d:\Prog\ScalaJS\NESemul\target\scala-2.11\nesemul-fastopt.js:1544:8)[0m
[0m[[31merror[0m] [0m    at $c_Ltutorial_webapp_HelloWorldApp$.main (d:\Prog\ScalaJS\NESemul\target\scala-2.11\nesemul-fastopt.js:1547:15)[0m
[0m[[31merror[0m] [0m    at Object.onload ([stdin]:59:97)[0m
[0m[[31merror[0m] [0m    at process.nextTick (D:\Prog\ScalaJS\NESemul\node_modules\jsdom\lib\jsdom.js:317:18)[0m
[0m[[31merror[0m] [0m    at _combinedTickCallback (internal/process/next_tick.js:67:7)[0m
[0m[[31merror[0m] [0m    at process._tickCallback (internal/process/next_tick.js:98:9)[0m
org.scalajs.jsenv.ExternalJSEnv$NonZeroExitException: Node.js with JSDOM exited with code 1
	at org.scalajs.jsenv.ExternalJSEnv$AbstractExtRunner.waitForVM(ExternalJSEnv.scala:107)
	at org.scalajs.jsenv.ExternalJSEnv$ExtRunner.run(ExternalJSEnv.scala:156)
	at org.scalajs.sbtplugin.ScalaJSPluginInternal$.org$scalajs$sbtplugin$ScalaJSPluginInternal$$jsRun(ScalaJSPluginInternal.scala:684)
	at org.scalajs.sbtplugin.ScalaJSPluginInternal$$anonfun$60$$anonfun$apply$43$$anonfun$apply$44.apply(ScalaJSPluginInternal.scala:771)
	at org.scalajs.sbtplugin.ScalaJSPluginInternal$$anonfun$60$$anonfun$apply$43$$anonfun$apply$44.apply(ScalaJSPluginInternal.scala:765)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
[0m[[31merror[0m] [0m(compile:[31mrun[0m) org.scalajs.jsenv.ExternalJSEnv$NonZeroExitException: Node.js with JSDOM exited with code 1[0m
